src/libs/ajax/data-table-providers/WdsDataTableProvider.ts

```diff
@@ -280,6 +281,10 @@ export class WdsDataTableProvider implements DataTableProvider {
 
   uploadTsv = (uploadParams: UploadParameters): Promise<TsvUploadResponse> => {
     if (!this.proxyUrl) return Promise.reject('Proxy Url not loaded')
+    const filesize = uploadParams.file?.size || Number.MAX_SAFE_INTEGER
+    if (filesize > 131072) {
+      notifyDataImportProgress(uploadParams.recordType, 'Your data will show up under Tables once import is complete.')
+    }
```

Comment:
instead of using a hardcoded size limit, would it be possible to make this progress notification be time-based using `setTimeout`? For instance, if the upload hasn't yet completed within 2 seconds, show the progress notification; otherwise, don't show it. This might be tricky code-wise because you'd need to avoid the race condition of showing the "success" notification at 1.9 seconds and then the "in progress" notification at 2 seconds.

Reply:
Alright I gave it a try - I think it shouldn't show the in progress notification after the success, but I can't promise the success notification won't come immediately after the in-progress.